0053-bundle-format/README.md

```diff
@@ -0,0 +1,474 @@
+# ZEP-0053: Sigstore Bundle Format Migration and Verification Strategy
+
+## Summary
+
+This proposal outlines the migration from Zarf's legacy signature format (`zarf.yaml.sig`) to the standardized Sigstore bundle format (`zarf.bundle.sig`) as the default signing mechanism. It establishes connectivity profiles for signing and verification operations, ensuring Zarf maintains its airgap-first philosophy while supporting online verification scenarios when connectivity is available.
+
+## Motivation
+
+Zarf currently supports the legacy signature format via asymmetrical keypairs, with incomplete handling of various connectivity scenarios and signing strategies:
+
+1. **Keypair-only support**: Signing currently only supports asymmentrical keypairs
+2. **Missing online verification path**: No support for transparency log verification when connectivity is available
+3. **Lack of keyless signing support**: Does not support keyless signing/verification
+4. **Lack of portability**: Public key distribution adds an additional artifact for every unique keypair that must be retrieved - zarf does not handle this natively
+
+These gaps create confusion for users deploying Zarf in different environments and may lead to security misconfigurations.
+
+### Goals
+
+- Migrate to Sigstore bundle format as the default signature format
+- Define explicit connectivity profiles (offline, online) with sensible defaults
+- Enable optional online verification with transparency log when connectivity is available
+- Maintain backward compatibility with offline keypair signing
+- Supporting keyless (OIDC-based) signing with online and private signing infrastructure
+
+### Non-Goals
+
+- Implementing a chain of custody for Zarf package verification
```

Comment:
This is a non-goal for sheer scope. I believe this is accomplishable but would build upon the scope of this proposal.